src/Akka.Persistence.Sql.Linq2Db/Journal/Types/JournalTagRow.cs

```diff
@@ -9,7 +9,8 @@ namespace Akka.Persistence.Sql.Linq2Db.Journal.Types;
 
 public sealed class JournalTagRow
 {
-    public long JournalOrderingId { get; set; }
-    
+    public long SequenceNumber { get; set; }
+    public string PersistenceId { get; set; }
+    public long OrderingId { get; set; }
```

Comment:
TagTable rows are expanded to include the event persistence id and sequence number. That way we can delete TagTable entries by following the same arguments as journal events deletion arguments (by using persistence id and sequence number)

Reply:
Makes sense - remember discussing this a while ago. This is something I wish we could have done with Akka.Persistence.Azure but the query model doesn't easily allow for it.